third_party/kiva-ep/src/libkiva/CMakeLists.txt

```diff
@@ -182,7 +182,7 @@ target_include_directories(libkiva PUBLIC
 target_compile_options(libkiva PRIVATE
   $<$<CXX_COMPILER_ID:MSVC>:/W4 /WX>
   $<$<OR:$<CXX_COMPILER_ID:Clang>,$<CXX_COMPILER_ID:AppleClang>,$<CXX_COMPILER_ID:GNU>>:
-  -Wall -Wextra -Wpedantic -Werror>
+  -Wall -Wextra -Wpedantic -Werror -Wno-uninitialized>
```

Comment:
Avoid a warning treated as an error in kiva / boost with gcc 11 and above

https://github.com/jmarrec/EnergyPlus/runs/7992086154?check_suite_focus=true


```
In file included from /home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/vendor/boost-1.77.0/boost/geometry/algorithms/detail/overlay/handle_colocations.hpp:36,
                 from /home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/vendor/boost-1.77.0/boost/geometry/algorithms/detail/overlay/enrich_intersection_points.hpp:39,
                 from /home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/vendor/boost-1.77.0/boost/geometry/algorithms/detail/buffer/buffered_piece_collection.hpp:51,
                 from /home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/vendor/boost-1.77.0/boost/geometry/algorithms/detail/buffer/buffer_inserter.hpp:29,
                 from /home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/vendor/boost-1.77.0/boost/geometry/algorithms/buffer.hpp:41,
                 from /home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/vendor/boost-1.77.0/boost/geometry/geometry.hpp:66,
                 from /home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/vendor/boost-1.77.0/boost/geometry.hpp:17,
                 from /home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/src/libkiva/Geometry.hpp:22,
                 from /home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/src/libkiva/Foundation.hpp:9,
                 from /home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/src/libkiva/Foundation.cpp:4:
/home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/vendor/boost-1.77.0/boost/geometry/algorithms/detail/overlay/sort_by_side.hpp: In member function 'Point boost::geometry::detail::overlay::sort_by_side::side_sorter<Reverse1, Reverse2, OverlayType, Point, SideStrategy, Compare>::add(const Turn&, const Operation&, boost::geometry::signed_size_type, int, const Geometry1&, const Geometry2&, bool) [with Turn = boost::geometry::detail::overlay::traversal_turn_info<boost::geometry::model::point<double, 2, boost::geometry::cs::cartesian>, boost::geometry::segment_ratio<double> >; Operation = boost::geometry::detail::overlay::traversal_turn_operation<boost::geometry::model::point<double, 2, boost::geometry::cs::cartesian>, boost::geometry::segment_ratio<double> >; Geometry1 = boost::geometry::model::box<boost::geometry::model::point<double, 2, boost::geometry::cs::cartesian> >; Geometry2 = boost::geometry::model::multi_polygon<boost::geometry::model::polygon<boost::geometry::model::point<double, 2, boost::geometry::cs::cartesian>, true, false> >; bool Reverse1 = false; bool Reverse2 = true; boost::geometry::overlay_type OverlayType = boost::geometry::overlay_difference; Point = boost::geometry::model::point<double, 2, boost::geometry::cs::cartesian>; SideStrategy = boost::geometry::strategy::side::side_by_triangle<void>; Compare = std::less<int>]':
/home/runner/work/EnergyPlus/EnergyPlus/third_party/kiva-ep/vendor/boost-1.77.0/boost/geometry/algorithms/detail/overlay/sort_by_side.hpp:299:15: error: 'point.boost::geometry::model::point<double, 2, boost::geometry::cs::cartesian>::m_values[0]' is used uninitialized [-Werror=uninitialized]
  299 |         Point point;
      |               ^~~~~
```